### DISCUSSION TITLE
フォントを変更するには？

### DISCUSSION THREAD

**Discussion by jasondee1992** [kmuto/review#1861] [Q&A]
特定の単語の日本語フォント スタイルを変更するにはどうすればよいですか?

例: タイトルのフォント スタイルを変更したいですか?

**Comment by kmuto** [comment]
サンプルを示すことができますか？
「特定」が本当に一部なのか、それともタイトル全部なのか、日本語文字ではなく実は欧文文字なのか によっても話が変わってきます。
そして、Re:VIEWとは関係がないことの可能性も考えられます。

実際のところ、 @jasondee1992 さんの過去の質問を見ていても、何をgoalにしているのかがはっきりしません。Re:VIEWの中に生のTeXをたくさん書いたり、review-custom.styにあちこちから拾ってきたものを書いて失敗するのを繰り返すのは、Re:VIEWの推奨する使い方ではなく、間違っている方向へ走っているようにしか思えません。
